libraries/resource_helpers.rb

```diff
@@ -17,6 +17,10 @@ def splunk_bin_path
       Pathname.new(install_dir).join('bin')
     end
 
+    def splunk_app_path
```

Comment:
I liked the idea of putting it next to the similar splunk_bin_path... but you're right, it's only going to be used in splunk_app, and it's probably going to disappear when clustering support is added.